src/dataflow/components/dataflow-program.tsx

```diff
@@ -57,21 +60,39 @@ export class DataflowProgram extends BaseComponent<IProps, IState> {
   constructor(props: IProps) {
     super(props);
     this.state = {
-      disableDataStorage: false
+      disableDataStorage: false,
+      programRunTime: ProgramRunTimes[3].val,
```

Comment:
I think it would be better to just hard-code this to `600` (or whatever). Otherwise, when we assign our newest engineer to implement the hypothetical one-point story to add a 10-second option to the list of available durations, the second story we assign to them will be to fix the default which was broken with their first commit. In other words, let's not couple the specification of the default duration to the decision about the set of options to show in the UI.

Reply:
Which also makes me realize to be complete I need to pass this to the component and set it in the dropdown.

Reply:
Fixed.